### ISSUE TITLE
Response code 406 even on passing correct credentials

### ISSUE THREAD

**Issue by Truptii** [imbo/behat-api-extension#76]
I am using BehatAPIExtentsion and writing a simple scenario to verify the response code for an endpoint, but I am getting error 406 on executing the scenario. This works properly on POSTMAN tool but the test on behat is failing.

I tried to pass the credentials using the following:
Scenario: Verify the login 
Given the request body is: 
"""
 { 
"username": "admin", 
"password": "qa123" 
}
 """ 
When I request "/api/v1/user/login" using HTTP "POST" 
Then the response code is "200"

 Scenario: Verify the login 
Given the following form parameters are set: 
| name | value | 
| username | admin | 
| password | qa123 | 
When I request "/api/v1/user/login" using HTTP "POST" Then the response code is 200

Note: Basic Authentication will not work here so did not use
 Given I am authenticating as "admin" with password "qa123"

Expected: The response code should be 200

Actual: The scenarios don't pass as the response code received is 406. It is not able to login using the credentials in the scenario. Error : Imbo\BehatApiExtension\Exception\AssertionFailedException: 

Any help would be appreciated.

**Comment by miteshmap** [comment]
@Truptii can you paste a working curl request here?

**Comment by Truptii** [comment]
Here is the curl request
curl -X POST \
  https://end-point-base-url/api/v1/user/login \
  -H 'cache-control: no-cache' \
  -H 'content-type: multipart/form-data;' \
  -F username=admin \
  -F password=qa123

**Comment by miteshmap** [comment]
Thanks for sharing. you are trying to post data as `multipart/form-data`, which is not supported 
 content-type without uploading a file.

**Comment by miteshmap** [comment]
Created #78 to support the multipart content type with post data and without any required file upload.

**Comment by christeredvartsen** [comment]
Should be resolved by #78.

@Truptii You should be able to use the following:

```gherkin
Scenario: Verify the login
    Given the following multipart form parameters are set:
        | name     | value |
        | username | admin |
        | password | qa123 |
    When I request "/api/v1/user/login" using HTTP "POST"
    Then the response code is 200
```

Notice the new step:

    Given the following multipart form parameters are set:
